fp_app/spec/models/user_spec.rb

```diff
@@ -0,0 +1,36 @@
+# spec/models/user_spec.rb
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
```

Comment:
```suggestion
RSpec.describe User, type: :model do
    describe 'test for devise validation' do
       it 'hoge ~' do
         aaaa
       end
    end
end
```
deviseのvalidationであることをわかりやすくする例

Reply:
ありがとうございます！
deviseのvalidationであることをわかりやすくするために修正しておきます。